Prace_domowe/Praca_domowa4/grudzien_adrianna/pd4.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>Z dokumentacji:</p><p> </p><p><strong>degree<em>int, default=3</em></strong></p><p>Degree of the polynomial kernel function (‘poly’). Ignored by all other kernels.</p><p> </p><p><strong>kernel<em>{‘linear’, ‘poly’, ‘rbf’, ‘sigmoid’, ‘precomputed’}, default=’rbf’</em></strong></p><p> </p><p>fitujesz nieużywany parametr =</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>